api/envoy/admin/v3/config_dump.proto

```diff
@@ -61,6 +62,14 @@ message UpdateFailureState {
   // This is the version of the rejected resource.
   // [#not-implemented-hide:]
   string version_info = 4;
+
+  // This resource is not included in the update or being explicitly removed by
+  // an update, either case means that this resource no longer existed in the
+  // xDS server. For more information, please refer to the :ref:`"Knowing When a
+  // Requested Resource Does Not Exist" <xds_protocol_resource_not_existed>`
```

Comment:
Where is this document?

Reply:
You can find the live version: https://www.envoyproxy.io/docs/envoy/latest/api-docs/xds_protocol#knowing-when-a-requested-resource-does-not-exist